### ISSUE TITLE
Todo preset prints warnings related to commons-logging.jar

### ISSUE THREAD

**Issue by tarekoraby** [vaadin/hilla#1372]
### Describe the bug

Running the [todo preset](https://start.vaadin.com/dl?preset=react-tutorial&projectName=hilla-todo) for the first time generates the following warning few times in the logs:

```
Standard Commons Logging discovery in action with spring-jcl: please remove commons-logging.jar from classpath in order to avoid potential conflicts
```

### Reproduction

Download and run https://start.vaadin.com/dl?preset=react-tutorial&projectName=hilla-todo

### System Info

Windows, Hilla 2.2, IntelliJ IDEA, JDK 20.

**Comment by Artur-** [comment]
Was this resolved in 2.3 alpha 6?

**Comment by tarekoraby** [comment]
It seems so, yes.
